### PR TITLE
Peaches solved 1431 

### PR DESCRIPTION
**Pull request by Peaches255**

### CODE REVIEW

File: problems/1431.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            maxNumberOfCandy = max(kid, maxNumberOfCandy)
+
+        output_list = [None]*len(candies)
+        for i in range(len(candies)):
+            output_list [i] = self.amongGreatestNumberOfCandies( candies[i], extraCandies, maxNumberOfCandy)
+
+        return output_list
+
+    def amongGreatestNumberOfCandies(self, candy, extraCandies,                 maxNumberOfCandy):
+        #function among greatest number of candies
```
Comment (chrischris292) [inline problems/1431.py:21]: Delete this

File: problems/1431.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        :type extraCandies: int
+        :rtype: List[bool]
+        """
+        #find the max number of candy
+        # for each element add extraCandies
+        maxNumberOfCandy = 0
+        for kid in candies:
+            maxNumberOfCandy = max(kid, maxNumberOfCandy)
+
+        output_list = [None]*len(candies)
```
Comment (chrischris292) [inline problems/1431.py:14]: It is helpful to just append for interview/leetcode problems to make debugging easier

Comment (chrischris292) [reply problems/1431.py:14]: But this is obviously the most clean solution so nice job!

File: problems/1431.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Solution(object):
+    def kidsWithCandies(self, candies, extraCandies):
+        """
+        :type candies: List[int]
+        :type extraCandies: int
+        :rtype: List[bool]
+        """
+        #find the max number of candy
+        # for each element add extraCandies
+        maxNumberOfCandy = 0
```
Comment (chrischris292) [inline problems/1431.py:10]: What about negatives? Have u considered using math.min instead? This is the smallest number integer supports. Negative candies doesn’t really make sense but I think should know this for future interviews

**Review by chrischris292 — APPROVED** [review]

**Review by chrischris292 — CHANGES_REQUESTED** [review]
Testing

**Review by chrischris292 — APPROVED** [review]

File: problems/1431.py (hunk 3 above)
Comment (Peaches255) [reply problems/1431.py:10]: I think u mean minint which python doesn't have. Python has this instead `-sys.maxint - 1` which I will use for future code

**Pull request merged**